src/i18n/Localizable.xcstrings

```diff
@@ -12627,6 +12703,7 @@
       }
     },
     "Lifetime_Purchase" : {
+      "comment" : "is this a \"lifetime purchase\", not \"purchase lifetime license\"?",
```

Comment:
You buy it once and can use it for your lifetime, updates are available for free without needing to purchase it again.